code/game/objects/machinery/overwatch.dm

```diff
@@ -67,6 +67,8 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	var/datum/action/innate/order/rally_order/send_rally_order
 	///Groundside minimap for overwatch
 	var/datum/action/minimap/marine/external/cic_mini
+	///Overrides the minimap action minimap and marker flags
+	var/map_flags = MINIMAP_FLAG_MARINE
```

Comment:
Did you want to override this on the SoM cams?

Reply:
ay.lmao.